recipe/meta.yaml

```diff
@@ -1,20 +1,19 @@
-{% set version = "3.20.1" %}
+{% set version = "21.4" %}
 
 package:
   name: protobuf
   version: {{ version }}
 
 source:
   url: https://github.com/protocolbuffers/protobuf/archive/v{{ version }}/protobuf-v{{ version }}.tar.gz
-  sha256: 8b28fdd45bab62d15db232ec404248901842e5340299a57765e48abe8a80d930
+  sha256: 990e47a163b4057f98b899eca591981b5b735872b58f59b9ead9cecabbb21a2a
 
 build:
   number: 0
   script:
     - cd python
     - >-
-        "{{ PYTHON }}" -m pip install . -vv
-        --install-option="--cpp_implementation"  # [unix]
+      "{{ PYTHON }}" -m pip install . -vv --install-option="--cpp_implementation"  # [unix]
```

Comment:
Removing the line break causes the `[unix]` selector to do nothing on Windows.
See also: https://github.com/conda-forge/protobuf-feedstock/pull/145/commits/ad24e71c8fa9c5747661e987db0fb23eca065109

Can we prevent the bot from doing this somehow?

Reply:
Maybe we can add a 

```
script_env:
 - EXTRA_BUILD_FLAGS=--install-option="--cpp_implementation" # [unix]
```

And then replace the script with

```
"{{ PYTHON }}" -m pip install . -vv $EXTRA_BUILD_FLAGS"
````

Reply:
Ping @xhochy. (I'm currently facing https://github.com/protocolbuffers/protobuf/issues/10283) which could be solved by this update.

Reply:
Note that there are also some version numbering concerns: https://github.com/conda-forge/protobuf-feedstock/pull/146#issuecomment-1149410254

Reply:
Alternatively we can maybe add some kind of dummy comment or a [True] selector on the first line to prevent the bot from merging the lines.